src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/AddSingleCaseToMultipleService.java

```diff
@@ -92,13 +90,25 @@ public void addSingleCaseToMultipleLogic(String userToken, CaseData caseData, St
 
     }
 
+    private String createMultipleReferenceLink(CaseData caseData)
+    {
+        var newMultipleReference = caseData.getMultipleReference() != null ?
```

Comment:
caseData.getMultipleReference() already is a String. There is no need for toString() here